src/Raven.Studio/wwwroot/App/views/database/tasks/editSqlEtlTask.html

```diff
@@ -312,7 +314,7 @@ <h3 class="text-center text-muted" data-bind="css: { 'text-danger': !transformat
                             <div class="form-group flex-horizontal">
                                 <label class="control-label">Document ID</label>
                                 <div class="flex-grow" data-bind="validationElement: documentId">
-                                    <input type="text" id="documentId" data-bind="textInput: documentId" class="form-control" placeholder="Document ID"
+                                    <input type="text" id="documentId" data-bind="textInput: documentId" class="form-control" placeholder="Document ID" autocomplete="off"
```

Comment:
don't we already have autocomplete off on entire form?

Reply:
there is autocomplete on entire 'form' which is on the left side of screen
this was missing on the right side